package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.80.0",
+  "version": "0.82.0",
```

Comment:
Assuming https://github.com/Health-Education-England/tis-trainee-ui/pull/937 is merged first, hence the version leap-frog :)